tests/TestBatch/test_compiler_batch.tcl

```diff
@@ -21,17 +21,17 @@ batch {
   pin_loc_assign_method random
   synth
   packing
-  globp
-  
+  # globp
```

Comment:
There's one test upstream that needs this update.

Reply:
tests/TestBatch/test_compiler_batch.tcl

Reply:
good catch, I've added it to the existing PR on _rs.